cmd/k8sviz/main.go

```diff
@@ -70,11 +69,13 @@ func init() {
 	}
 
 	// test connectivity for k8s cluster and the namespace
-	_, err = clientset.CoreV1().Namespaces().Get(namespace, metav1.GetOptions{})
-	if err != nil {
-		fmt.Fprintf(os.Stderr, "Failed to get namespace %q: %v\n", namespace, err)
-		os.Exit(1)
-	}
+	//if !strings.Contains(namespace,",") {
```

Comment:
If namespaces are parsed into the slice(see below), we will be able to check for all the namespaces here.

Reply:
By "below", I meant [this comment](https://github.com/mkimuram/k8sviz/pull/26#discussion_r606325642). It appeared above this comment :)